sympy/holonomic/holonomic.py

```diff
@@ -786,6 +787,31 @@ def composition(self, expr):
             return HolonomicFunction(sol, self.x, self.x0, self.y0)
         return HolonomicFunction(sol, self.x)
 
+    def to_Sequence(self):
```

Comment:
I think this should be `to_sequence` according to [SymPy conventions](https://github.com/sympy/sympy/wiki/Development-workflow#id23).